src/project/manifest/pyproject.rs

```diff
@@ -21,8 +20,16 @@ pub struct PyProjectManifest {
 }
 
 #[derive(Deserialize, Debug, Clone)]
-struct Tool {
-    pixi: ProjectManifest,
+pub struct Tool {
+    pub pixi: Option<ProjectManifest>,
+    #[allow(dead_code)]
+    pub poetry: Option<ToolPoetry>,
```

Comment:
Cant we just remove this?

Reply:
it will be used in PR where we upgrade uv for this:

```rust
// // If `[project]` is not present, we assume it's dynamic.
let Some(project) = pyproject_toml.project else {
        // ...unless it appears to be a Poetry project.
        return pyproject_toml
            .tool
            .map_or(true, |tool| tool.poetry.is_none());
    };
```

Reply:
👍